pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/ElasticDataExportAdapter.java

```diff
@@ -18,6 +19,7 @@ public void export() {
         logger.info("initialize exporting data to ES");
         UpdateRequest updateRequest = new UpdateRequest(requestPayload.getIndex(), Util.DOC, requestPayload.getDocId())
                 .doc(source)
+                .setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE)
```

Comment:
We'll need to monitor the impact of this change.  From the [docs](https://www.javadoc.io/doc/org.elasticsearch/elasticsearch/6.8.15/org/elasticsearch/action/support/WriteRequest.RefreshPolicy.html) with my emphasis added:
Force a refresh as part of this request. **This refresh policy does not scale for high indexing or search throughput** but is useful to present a consistent view to for indices with very low traffic. And it is wonderful for tests!

Reply:
Yes. See my note in the PR description. At this point in our cycle it might make sense to add a separate Exportable class just for onc history upload. 
I don't *think* we see this type of issue elsewhere in DSM, but that may be because we make the changes in-place in the UI, and therefore do not need the index to be up to date immediately?
First, let's see if this fixes the bug.